annotate.el

```diff
@@ -871,6 +879,14 @@ essentially what you get from:
 (annotate-annotations-from-dump (annotate-load-annotations))). "
   (nth 2 annotation))
 
+(defun annotate-sample-text-of-annotation (annotation)
```

Comment:
These function names are confusing. I think we should find a better naming scheme for `text-of-annotation` and `sample-text-of-annotation`. 

Maybe `annotate-destination-text-of-annotation` and `annotate-annotation-text-of-annotation`?

It's not perfect yet...

Reply:
So if i understand correctly:

``` text
 from 'annotate-sample-text-of-annotation' to  'annotate-annotation-text-of-annotation'
 from 'annotate-text-of-annotation'        to  'annotate-destination-text-of-annotation'
```
right?

Reply:
Yeah, but now that you write them out, those are not good names yet.

So we have to differentiate between the text being annotated, and the annotation text. Maybe that's the key right there: "annotation text" vs. "annotated text". And maybe mix up that "text" to make them more distinct:

    from annotate-sample-text-of-annotation   to annotate-annotated-text
    from annotate-text-of-annotation          to annotate-annotation-string

(And possibly reword other occurrences accordingly)

What do you think about that?

Reply:
Hi bastibe, and welcome back! :-)

I agree with the changes and i am going to push changes accordingly to your suggestions. I have to admit i am a bit confused between annotated text and annotation text i hope i did right with this changes. :-D

Bye!
 C.